src/modules/vtol_att_control/standard.cpp

```diff
@@ -368,6 +368,12 @@ void Standard::fill_actuator_outputs()
 
 	_actuators_out_1->control[actuator_controls_s::INDEX_THROTTLE] = _pusher_throttle;
 
+	/* translate MC actuators to FW actuators */
+	if (_vtol_schedule.flight_mode == TRANSITION_TO_FW || _vtol_schedule.flight_mode == TRANSITION_TO_MC) {
+		_actuators_out_1->control[actuator_controls_s::INDEX_ROLL] = -_actuators_mc_in->control[actuator_controls_s::INDEX_YAW];
+		_actuators_out_1->control[actuator_controls_s::INDEX_PITCH] = _actuators_mc_in->control[actuator_controls_s::INDEX_THROTTLE] - 0.5f;
```

Comment:
A) The pitch commanded during transition should only be based on holding altitude. The MC controller actuates this through throttle, not pitch.
B) cg is not the only factor, also, the pitch attitude is not constant as you can see from the logs.

Your suggestion would work only in one direction but it would indeed work. I will test fly my suggestion today and we can see further based on those logs.